src/pages/settings/index.jsx

```diff
@@ -50,95 +52,103 @@ export default function Settings() {
             labelPlacement="end"
-            sx={{
-              marginInlineStart: `${2 * (pageUrl.split("/").length - 1)}px`,
-            }}
           />
-        ))}
-      {/* color mode */}
-      <Button
-        sx={{ ml: 1 }}
-        onClick={colorMode.toggleColorMode}
-        color="inherit"
-      >
-        {theme.palette.mode} mode
-        {theme.palette.mode === "dark" ? (
-          <Brightness7Icon sx={{ ml: 2 }} />
-        ) : (
-          <Brightness4Icon sx={{ ml: 2 }} />
-        )}
-      </Button>
-
-      <div>
-        <MuiLink
-          href="https://github.com/openfoodfacts/hunger-games/issues"
-          target="_blank"
+        </div>
+        {devMode &&
+          [
+            { pageUrl: "logos", pageName: "logos", isExperimental: false },
+            {
+              pageUrl: "logos/search",
+              pageName: "search logos",
+              isExperimental: false,
+            },
+            {
+              pageUrl: "logos/product-search",
+              pageName: "search product logos",
+              isExperimental: true,
+            },
+            {
+              pageUrl: "logos/deep-search",
+              pageName: "deep search logo",
+              isExperimental: true,
+            },
+            {
+              pageUrl: "dashboard",
+              pageName: "dashboard",
+              isExperimental: false,
+            },
+            {
+              pageUrl: "insights",
+              pageName: "insights",
+              isExperimental: false,
+            },
+          ].map(({ pageUrl, pageName, isExperimental }) => (
+            <FormControlLabel
+              key={pageUrl}
+              checked={visiblePages[pageUrl] ?? false}
+              onChange={handleVisiblePagesChange(pageUrl)}
+              control={<Switch />}
+              label={`${t("settings.dev_page_toggle", { name: pageName })}${
+                isExperimental ? " (🚧 experimental)" : ""
+              }`}
+              labelPlacement="end"
+              sx={{
+                marginInlineStart: `${2 * (pageUrl.split("/").length - 1)}px`,
+              }}
+            />
+          ))}
+        {/* color mode */}
+        <Button
+          sx={{ ml: 1 }}
+          onClick={colorMode.toggleColorMode}
+          color="inherit"
         >
-          {t("settings.reportIssue")}
-        </MuiLink>
-      </div>
-    </Stack>
+          {theme.palette.mode} mode
+          {theme.palette.mode === "dark" ? (
+            <Brightness7Icon sx={{ ml: 2 }} />
+          ) : (
+            <Brightness4Icon sx={{ ml: 2 }} />
+          )}
+        </Button>
+
+        <div>
+          <MuiLink
+            href="https://github.com/openfoodfacts/hunger-games/issues"
+            target="_blank"
+          >
+            {t("settings.reportIssue")}
+          </MuiLink>
+        </div>
+      </Stack>
+      <Divider sx={{ width: "100%" }} light />
+      <FooterWithLinks />
```

Comment:
Very nice

Maybe a slight improvement could be to move all the files you created to a folder `src/components/Footer` such that you can also reuse it in `src/pages/home`

Reply:
okayyyyy !!!! I will do that. Thanks <3